lv2/log/logger.h

```diff
@@ -114,7 +114,7 @@ lv2_log_error(LV2_Log_Logger* logger, const char* fmt, ...)
 /** Log a note via lv2_log_vprintf(). */
 LV2_LOG_FUNC(2, 3)
 static inline int
-lv2_log_note(LV2_Log_Logger* logger, const char* fmt, ...)
+lv2_log_note(const LV2_Log_Logger* logger, const char* fmt, ...)
```

Comment:
I don't know, this seems pretty weird to me.  Even though it happens to work, because `LV2_Log_Log` has a non-const handle which is the "actual" thing being logged to, it seems misleading to have this parameter be const.  Writing to something isn't const.

I tend to prefer faking "transitive constness".  For example, if this were C++, it definitely wouldn't be const, because the `printf` and `vprintf` methods aren't const.  You just happen to be able to make the `LV2_Log_Log` struct itself const, because it's faking OO with the handle.